### PR TITLE
Create imports for wpilib

### PR DESCRIPTION
**Pull request by TheTripleV**

### CODE REVIEW

File: robotpy-wpilib/wpilib/__init__.py [hunk 1]
```diff
@@ -1,6 +1,417 @@
...
+    getCurrentThreadPriority,
+    getDeployDirectory,
+    getLaunchDirectory,
+    getOperatingDirectory,
+    getThreadPriority,
+    getTime,
+    setCurrentThreadPriority,
+    setThreadPriority,
+    wait,
+    wpi_error_s_AnalogTriggerLimitOrderError,
```
Comment (virtuald) [inline robotpy-wpilib/wpilib/__init__.py:96]: whoa, I wonder where these came from

File: robotpy-wpilib/wpilib/__init__.py [hunk 2]
```diff
@@ -1,6 +1,417 @@
...
+    DigitalSource,
+    DoubleSolenoid,
+    DriverStation,
+    DutyCycle,
+    DutyCycleEncoder,
+    Encoder,
+    Error,
+    ErrorBase,
+    GyroBase,
+    HAL_GetErrorMessage,
```
Comment (auscompgeek) [inline robotpy-wpilib/wpilib/__init__.py:40]: wait why is this showing up here

Comment (TheTripleV) [reply robotpy-wpilib/wpilib/__init__.py:40]: `... import *` works just fine but pylint doesn't detect anything and puts red squigglies everywhere

Comment (auscompgeek) [reply robotpy-wpilib/wpilib/__init__.py:40]: no I mean why are bits of the HAL leaking into our wpilibc bindings

**Pull request merged**